### ISSUE TITLE
Allow dots in blocks

### ISSUE THREAD

**Issue by limonte** [levibuzolic/eslint-plugin-no-only-tests#14]
Currently, the plugin doesn't work for this code:

```js
ava.default.only(...)
```

configuration:
```js
  rules: {
    'no-only-tests/no-only-tests': ['error', {
      block: ['ava.default'],
      focus: ['only']
    }]
  }
```

It seems that `.` aren't supported in blocks.

**Comment by levibuzolic** [comment]
Thanks @limonte I'll take a look and see what I can do to make it work for this use case.

**Comment by levibuzolic** [comment]
@limonte should be fixed in [v2.4.0](https://github.com/levibuzolic/eslint-plugin-no-only-tests/releases/tag/v2.4.0) mind giving it a go and checking it works for you?

**Comment by limonte** [comment]
Well done, thank you @levibuzolic !
